uefi-test-runner/src/proto/misc.rs

```diff
@@ -0,0 +1,72 @@
+use uefi::prelude::*;
+use uefi::proto::misc::ResetNotification;
+use uefi::table::runtime;
+
+///
+/// you may see those log, it's nothing just for your computer firmware does not support the new UEFI feature of Timestamp Protocol.
```

Comment:
Can drop this whole comment block, since timestamp isn't being used here anymore.